portfolio/src/main/webapp/script.js

```diff
@@ -123,6 +123,17 @@ $("#filter-none").click(() => filterPhotos());
 $("#newest").click(() => sortPhotos("newest"));
 $("#oldest").click(() => sortPhotos("oldest"));
 
+/**
+ * fetch text from /data to display
+ */
+function getFetchRequest() {
+  fetch("/data").then((response) =>
+    response.text().then((txt) => $("#fetch-response").text(txt))
```

Comment:
maybe `append(txt)` or `replaceWith(txt)` would be better since you're sending an h1 header.